tests/posix/env/src/env.c

```diff
@@ -0,0 +1,180 @@
+/*
+ * Copyright (c) 2023, Meta
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <stdio.h>
+#include <stdlib.h>
+
+#include <zephyr/ztest.h>
+
+#define _m_home "/home/zephyr"
```

Comment:
Wasn't there a rule to avoid using underscore? Do we really need underscore here at all?

Reply:
there are rules about underscores, bit this doesn't violate any of them. a single underscore followed by a lowercase usually means "only usable within current scope"

Reply:
I see, imho these various underscore rules are really confusing so I try to avoid using them. Anyway, was just wondering about this.

Reply:
Fixed!